src/deploy.sh

```diff
@@ -142,6 +142,29 @@ function kernel_deploy()
   cleanup
 }
 
+# This function return the errors outputed by getopt, formatted to be
+# printed by kw
+function getopt_errors()
```

Comment:
Actually, this only makes sense in the context of deploy, since there is this error managing system here. For example, we remove the prefix `unrecognized option` only because `kernel_deploy` will Add "Invalid option" later: See line 57:

```
  parse_deploy_options "$@"
  if [[ "$?" -gt 0 ]]; then
    complain "Invalid option: ${options_values['ERROR']}"
    exit 22 # EINVAL
  fi
```